nucypher/characters/control/emitters.py

```diff
@@ -39,7 +39,7 @@ def message(self,
         self.log.debug(message)
 
     def echo(self,
-             message: str,
+             message: str = None,
```

Comment:
Allowing for an empty call to add a newline?

Reply:
Yep, like `print()`.